### DISCUSSION TITLE
Adding extra package to flet

### DISCUSSION THREAD

**Discussion by guillermollopis** [flet-dev/flet#1335] [Q&A]
### Question

I would like to use a package that is available in Flutter but not in Flet. Is there a way of adding the package to Flet and use it in Python?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
You either have to implement it yourself and then make a PR to the flet repo, or open an issue suggesting an implementation by flet devs.
